cnxarchive/transforms/converters.py

```diff
@@ -29,7 +30,19 @@
 
 
 def _gen_xsl(f, d=XSL_DIR):
-    return etree.XSLT(etree.parse(os.path.join(d, f)))
+    transform = etree.XSLT(etree.parse(os.path.join(d, f)))
+
+    def transform_w_version(*args, **kwargs):
+        # If git is not in $PATH, rhaptos.cnxmlutils.__version__ returns
+        # "0+unknown"
+        if '/usr/bin' not in os.getenv('PATH', ''):
+            os.environ['PATH'] = '/usr/bin:{}'.format(os.environ['PATH'])
+            reload(rhaptos.cnxmlutils)
```

Comment:
Interesting touch. 😸

Reply:
I read that this only works in python2... :p Will need to change it when we move to python3.

Reply:
Well, I just re-used it in the test for this feature, so two places to fix it for py3 :-(